packages/ERTP/src/amountMath.js

```diff
@@ -158,7 +158,7 @@ const checkLRAndGetHelpers = (leftAmount, rightAmount, brand = undefined) => {
  * @param {MathHelpers<AssetValueForKind<K>>} h
  * @param {Amount<K>} leftAmount
  * @param {Amount<K>} rightAmount
- * @returns {[K, K]}
+ * @returns {[AssetValueForKind<K>, AssetValueForKind<K>]}
```

Comment:
I reviewed before the force push with this. Does this not fix something?

Reply:
It does fix the type. When I added this, I also added to the PR description "and one internal type correction". This function does not return a pair of AssetKinds, it return a pair of values.

Reply:
Just now, I also changed the PR title to indicate it is a refactor to ertp and zoe.